bindings/python/tools/python/ostk/astrodynamics/display.py

```diff
@@ -183,36 +186,108 @@ def _append_line(
             )
         )
 
-    def add_satellite(
-        self, trajectory: Trajectory, accesses: list[Access], rgb: list[int]
+    def add_ground_station(
+        self,
+        ground_station_lla: LLA,
+        color: str,
     ) -> None:
         """
-        Add a satellite trajectory to the plot, including a highligh ot the interference
-        accesses.
-        """
+        Add a ground station to the plot.
 
-        # Satellite trajectory
-        satellite_trajectory_data: list[list[float]] = []
-        self._generate_and_append_data(
-            satellite_trajectory_data, trajectory, self._trajectory_grid
-        )
-        satellite_trajectory_df: pd.Dataframe = pd.DataFrame(
-            satellite_trajectory_data, columns=["Longitude", "Latitude"]
+        Args:
+            ground_station_lla (LLA): The ground station location.
+            color (str): The color of the ground station.
+        """
+        self._data.append(
+            dict(
+                type="scattergeo",
+                lon=[float(ground_station_lla.get_longitude().in_degrees())],
+                lat=[float(ground_station_lla.get_latitude().in_degrees())],
+                mode="markers",
+                marker=dict(
+                    size=10,
+                    color=color,
+                ),
+            )
         )
-        self._append_line(satellite_trajectory_df, 1, rgb, 0.1)
 
-        # Satellite accesses
+    def add_accesses(
+        self,
+        trajectory: Trajectory,
+        accesses: list[Access],
+        rgb: list[int],
```

Comment:
shouldn't we align the arguments and just have the `color` argument everywhere (and depracate rgb)?

Reply:
also have the color argument be nullable everywhere so it just chooses random colors if you don't want to supply it?